specification/fist/resource-manager/Microsoft.IoTFirmwareDefense/preview/2023-02-08-preview/common.json

```diff
@@ -0,0 +1,182 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-02-08-preview",
+    "title": "fist",
+    "description": "The definitions and parameters in this swagger specification will be used to manage the IoT Firmware Defense resources."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {},
+  "definitions": {
+    "UrlToken": {
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "SAS URL for creating or accessing a blob file.",
+          "type": "string",
+          "format": "uri",
+          "readOnly": true,
+          "x-nullable": true
```

Comment:
> "x-nullable": true [](http://example.com/codeflow?start=9&length=19)

As a general rule we don't recommend using x-nullable, as you can just omit the property from your payload, unless you *actually need to return 'null' from your API for some reason*.

Reply:
In this case, what does it mean when you return null?

Reply:
Ah I see. if omission is allowed that satisfies the backwards compatibility so I will remove the nullable property